src/HttpClient/Response/HttpResponse.php

```diff
@@ -28,12 +29,15 @@ class HttpResponse implements ResponseInterface
      */
     private $document;
 
+    /** @var PsrResponseInterface */
```

Comment:
This has to be nullable in the docs as well.

Reply:
Yes